src/components/ConfirmationPage/ConfirmationPage.js

```diff
@@ -0,0 +1,298 @@
+									return (
+										<option
+											style={{ color: "#000", background: "#FFF" }}
+											value={doc.value}
+											key={index}
+										>
+											{doc.option}
+										</option>
+									);
+								})}
+							</Select>
+						</Flex>
+						<Flex align='center' wrap='wrap' my={2}>
+							<Text
+								color={textColorLight[mode]}
+								my={2}
+								w={["100%", "150px", "200px", "200px"]}
+							>
+								Controller Account
+							</Text>
+							<Select
+								placeholder='Select Account'
+								w={[
+									"calc(100%)",
+									"calc(100% - 150px)",
+									"calc(100% - 200px)",
+									"calc(100% - 200px)"
+								]}
+							>
+								{props.controllerOptions.map((doc, index) => {
+									return (
+										<option
+											value={doc.value}
+											key={index}
+											style={{ color: "#000", background: "#FFF" }}
+										>
+											{doc.option}
+										</option>
+									);
+								})}
+							</Select>
+						</Flex>
+						<Flex align='center' wrap='wrap' my={2}>
+							<Text
+								color={textColorLight[mode]}
+								my={2}
+								w={["100%", "150px", "200px", "200px"]}
+							>
+								Risk Preference
+							</Text>
+							<Badge
+								mx={1}
+								px={3}
+								py={1}
+								fontSize='sm'
+								variantColor={getRiskLevelColor(props.riskPreference)}
+							>
+								{getRiskLevel(props.riskPreference)}
+							</Badge>
+						</Flex>
+						<Box w='100%' p={2} mt={6} h='50vh' overflow='auto'>
+							{props.validatorsList.map((validator, index) => {
+								return (
+									<ValidatorTile
+										key={index}
+										name={validator.name}
+										amount={validator.amount}
+										currency={props.currency}
+										avatar={validator.avatar}
+										colorMode={props.colorMode}
+									/>
+								);
+							})}
+						</Box>
+						<Text
+							textAlign='center'
+							color={textColorLight[mode]}
+							my={4}
+							fontSize='xs'
+						>
+							* the estimated daily earnings are indicative. Actual rewards may
+							vary.
+						</Text>
+						<Box
+							p={4}
+							my={4}
+							w='100%'
+							bg={messageBoxColors[mode].bg}
+							borderColor={messageBoxColors[mode].border}
+							borderWidth='1px'
+							rounded='md'
+						>
+							<Text
+								fontSize='xs'
+								color={messageBoxColors[mode].text}
+								textAlign='center'
+							>
+								Fees of 10.000 milli KSM will be applied to the submission and
+								funds will be locked for {props.eras} eras
+							</Text>
+						</Box>
+						<Flex align='flex-start'>
+							<Checkbox
+								mx={4}
+								my={1}
+								variantColor='blue'
+								size='lg'
+								isChecked={termsCheck}
+								onChange={e => {
+									setTermsCheck(!termsCheck);
+								}}
+							></Checkbox>
+							<Text fontSize='sm' color={textColorLight[mode]}>
+								I understand that the funds will be bonded, meaning the tokens
```

Comment:
For mobile this text seems too much, would you agree that we could instead have something like a "learn more" button to point users to the appropriate resource to learn about the risks? Hence, reducing the required text to something along the following lines: "I understand that the funds will be locked and am aware of the risks involved with staking."

Reply:
I believe this being a "disclaimer" text, it needs to be prominent (especially since these are just a couple of lines and not a whole page). We can probably reduce the font size for mobile, to save space, but we should let the whole text be visible.